tests/test_idpi/test_crop.py

```diff
@@ -0,0 +1,25 @@
+# Third-party
+from numpy.testing import assert_equal
+
+# First-party
+from idpi.grib_decoder import GribReader
+from idpi.operators import crop
+
+
+def test_crop(data_dir):
```

Comment:
I would check the values of all keys related to the supported grid definition templates.

Reply:
I'm checking the geography namespace as defined by earthkit data. If that's not sufficient I could decode the message to extract all of the relevant keys.

Reply:
Since you also change the numberOfDataPoints in the crop operator, you should at least also check that key.

Reply:
It's not part of the geography namespace so I need to decode the message to read it.

Reply:
OK, then let's stay with the test as it is now, and have a look at ECMWF's namespace concept later on.